CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.10.7 - 2024-08-08
+
+### Changed
+
+- Support the new unified testing binary format released with Xcode 15 Beta 5
```

Comment:
actually wait, is 16?

Reply:
It'll be 16. Is this format only used on macOS?

Reply:
@adam-fowler It is used on all platforms. On macOS we need to use the swiftpm-testing-helper binary but on windows/linux we can just invoke the unified binary directly.